<launch>
	<node name="spline_optimization_example" pkg="ewok_optimization" type="spline_optimization_example3" output="screen">
 		<!--<rosparam param="x_pos">[0.5, 15.0, 0.5]</rosparam> 
		<rosparam param="y_pos">[0.0, 0.0, 0.0]</rosparam> 
		<rosparam param="z_pos">[1.0, 1.0, 2.0]</rosparam>-->

		<!--
		<rosparam param="x_pos">[0.0, 5.0, 5.0, 10.0, 10.0]</rosparam> 
		<rosparam param="y_pos">[0.0, 0.0, 5.0, 5.0, 10.0]</rosparam> 
		<rosparam param="z_pos">[5, 5, 5, 8, 5, 8]</rosparam>
		-->

		<rosparam param="x_pos">[0.0, 0.0, 0.0, 5.0, 8.0, 8.0, -15.0, -15.0, 0.0]</rosparam> 
		<rosparam param="y_pos">[0.0, 0.0, 5.0, 5.0, 8.0, 8.0, 10.0, -5.0, 0.0]</rosparam> 
		<rosparam param="z_pos">[5.0, 3.0, 3.0, 5.0, 5.0, 10.0, 12.0, 12.0, 10.0]</rosparam>


		<param name="number_of_target" type="int" value="9"/>
	</node>
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ewok_optimization)/rviz/optimization_example.rviz" />

		<node pkg="tf" type="static_transform_publisher" name="world_to_map" args="0 0 0 0 0 0 world map 100" />
</launch>
